include/alp/reference/matrix.hpp

```diff
@@ -659,11 +675,13 @@ namespace alp {
 			public:
 
 				MatrixFunctor(
+					const bool initialized,
```

Comment:
Perhaps annotate that this is a temporary solution. A specific issue will be open about linking the status to input containers.